docs/paper/dev/getting-started/userdev.mdx

```diff
@@ -103,3 +109,27 @@ tasks.assemble {
 }
 ```
 
+## 1.20.5
+As of 1.20.5 Paper will be shipping with a Mojang-mapped runtime instead of reobfuscating the server to Spigot mappings as we have traditionally done since Paper 1.17.
```

Comment:
```suggestion
As of 1.20.5, Paper will be shipping with a Mojang-mapped runtime instead of reobfuscating the server to Spigot mappings as we have traditionally done since Paper 1.17.
```

also, hasn't the server been Spigot-mapped ever since Paper was created, not only since 1.17?